pages/mesosphere/dcos/services/cassandra/2.4.0-3.0.16/index.md

```diff
@@ -3,7 +3,7 @@ layout: layout.pug
 navigationTitle: Cassandra 2.4.0-3.0.16
 excerpt: DC/OS Apache Cassandra is an automated service that makes it easy to deploy and manage Apache Cassandra on DC/OS.
 title: Cassandra 2.4.0-3.0.16
-menuWeight: -1
+menuWeight: -2
```

Comment:
I think this is supposed to be `2`, not negative 2 because we want C* 2.6.0- shown first and its current weight is `1`